javascript-data-model/order-history.js

```diff
@@ -0,0 +1,78 @@
+var orderHistory = [
+  {
+    book: {
```

Comment:
What if they order more than one book? I don't think naming the property after the type of item is the right way to go, you should store all items together in one space, and elaborate on the type of item alongside the data that describes the item itself.